OA-55589 rejectUnauthorized option (#50)
Dynatrace/agent-nodejs: README.md

  require('@dynatrace/oneagent')({
    environmentid: '<environmentid>',
    apitoken: '<paastoken>',
    endpoint: '<endpoint url>', // specify endpoint url - not needed for SaaS customers
    rejectUnauthorized: true, // default is false which allows to use a self-signed certificate, see more below
    port: 8443 // specify port - default is 443
  });
} catch (err) {
  console.log('Failed to load OneAgent: ', err);
}
```

> The `rejectUnauthorized` option controls TLS certificate verification for the initial connection to `/v1/deployment/installer/agent/connectioninfo`.  

Reviewer: ```suggestion
The `rejectUnauthorized` option controls TLS certificate verification for the initial request to `/v1/deployment/installer/agent/connectioninfo`.  It has no impact to other OneAgent code module communication nor the monitored application.
```

  require('@dynatrace/oneagent')({
    environmentid: '<environmentid>',
    apitoken: '<paastoken>',
    endpoint: '<endpoint url>', // specify endpoint url - not needed for SaaS customers

>     options: { rejectUnathorized: true } // for self-signed certificate use `rejectUnathorized: false`

Reviewer 2: ```suggestion
    options: { rejectUnathorized: true } // for a self-signed certificate use `rejectUnathorized: false`
```

>     options: { rejectUnathorized: true } // for self-signed certificate use `rejectUnathorized: false`

Reviewer: ```suggestion
    options: { rejectUnauthorized: true } // for self-signed certificate use `rejectUnauthorized: false`
```

Author: Thanks for catching this, I have updated the commit.

>     options: { rejectUnathorized: true } // for self-signed certificate use `rejectUnathorized: false`

Reviewer: I think this requires more docs to avoid confusions.
We should clearly state that this setting effects only the single request to `/v1/deployment/installer/agent/connectioninfo` and nothing else.
We should also mention the default if not used.

Author: I have added more verbose description below the example usage for the new `rejectUnauthorized` option.
